### DISCUSSION TITLE
como actualizo proyecto anterior

### DISCUSSION THREAD

**Discussion by DiegoBal** [WhatsApp/stickers#772] [Q&A]
hola buen día, anteriormente tenia un proyecto, una aplicación ya creada con stickers normales. quisiera no perder ese proceso agregando los nuevos stickers animados, será que puedo hacerlo? sin perder todo lo demás …?

**Comment by simonzhexu** [comment]
Please use English here.

**Reply by DiegoBal** [reply]
hello good day, previously I had a project, an application already created with normal stickers. I would like not to lose that process by adding the new animated stickers, can I do it? without losing everything else ...?

**Reply by yosoyjayakumar** [reply]
Hi Diego

You Lose your Old App Code and Your Changes You Simply Add That Animated Features in Your Existing Code open this Link

https://github.com/WhatsApp/stickers/commit/3c039d739708e24c12b936012eefc8fc22eb0566

Split Your Display Side by side Changes Are Showen in This Commit you Just Remove the Removed code add the Added code your App Code Then Build☺️

Any Doubt Contact me I will Help You

This is my App look This😉

https://play.google.com/store/apps/details?id=com.lowpow.tamiltrollstickers
